src/components/transactions/HexEncodedData/index.tsx

```diff
@@ -21,30 +22,27 @@ export const HexEncodedData = ({ hexData, title, limit = 20 }: Props): ReactElem
     setShowTxData((val) => !val)
   }
 
-  const firstBytes = <b>{hexData.slice(0, FIRST_BYTES)}</b>
+  const firstBytes = (
+    <Tooltip title="The first 4 bytes determine the method that gets called" arrow>
+      <b style={{ fontFamily: 'monospace' }}>{hexData.slice(0, FIRST_BYTES)}</b>
```

Comment:
We haven’t being using mono space anywhere else yet. It requires a special DM Mono font to look nice. I wouldn’t start here, it looks OK in sans.